OrcScala/src/orc/lib/progswap/ProgSwap.scala

```diff
@@ -30,6 +30,7 @@ import orc.error.runtime.{ ArgumentTypeMismatchException, ArityMismatchException
 import orc.run.core.Execution
 import orc.run.core.Token
 import orc.run.core.ExternalSiteCallHandle
+import orc.util.ArrayExtensions._
```

Comment:
Why the wildcard import?  Are there implicits used from here?  It looks like just one explicit import will do.